papers/srinath/article.md

```diff
@@ -1,15 +1,15 @@
 ---
 title: "An Analysis on Genomic Correlation for Gallstone Susceptibility"
 abstract: |
-  Gallstones, typically benign and harmless hardened deposits of digestive fluids, present in the gallbladder can cause severe painful complications if left untreated and can lead to removal surgery. (@noauthor_gallstones_2021) Quantitative Trait Loci (QTL) analysis can be used to find potential genetic links through the analysis of logarithm of the odds (LOD) scores which can indicate a possible connection between those loci on the mouse chromosome and phenotypic presentation of a trait (@myles2008quantitative) linked to gallstone susceptibility such as weight, presence, severity and liver weight. Analyses were performed using the R/QTL package on a cohort of mice, in an intercross breed and fed a high-fat diet. (@lyons1) Analyses were compared between augmented data sets (to possibly prevent overfitting) and an analysis run on a non-augmented data set.
+  Gallstones, typically benign and harmless hardened deposits of digestive fluids, present in the gallbladder can cause severe painful complications if left untreated and can lead to removal surgery. [@noauthor_gallstones_2021] Quantitative Trait Loci (QTL) analysis can be used to find potential genetic links through the analysis of logarithm of the odds (LOD) scores which can indicate a possible connection between those loci on the mouse chromosome and phenotypic presentation of a trait [@myles2008quantitative] linked to gallstone susceptibility such as weight, presence, severity and liver weight. Analyses were performed using the R/QTL package on a cohort of mice, in an intercross breed and fed a high-fat diet. [@lyons1] Analyses were compared between augmented data sets (to possibly prevent overfitting) and an analysis run on a non-augmented data set.
```

Comment:
I think this is included in the sentence before!? Up to you though. It is strange to start a sentance with a parenthetical citation.

```suggestion
  Gallstones, typically benign and harmless hardened deposits of digestive fluids, present in the gallbladder can cause severe painful complications if left untreated and can lead to removal surgery. [@noauthor_gallstones_2021] Quantitative Trait Loci (QTL) analysis can be used to find potential genetic links through the analysis of logarithm of the odds (LOD) scores which can indicate a possible connection between those loci on the mouse chromosome and phenotypic presentation of a trait [@myles2008quantitative] linked to gallstone susceptibility such as weight, presence, severity and liver weight. Analyses were performed using the R/QTL package on a cohort of mice, in an intercross breed and fed a high-fat diet [@lyons1]. Analyses were compared between augmented data sets (to possibly prevent overfitting) and an analysis run on a non-augmented data set.
```